src/forms/toggle/Toggle.tsx

```diff
@@ -49,7 +50,7 @@ export class Toggle extends React.PureComponent<ToggleProps, ToggleState> {
         }
     }
     render() {
-        const {label, name, ariaLabelledby, disabled} = this.props;
+        const {label, name, ariaLabelledby, disabled, onClick} = this.props;
```

Comment:
place the destruction to different lines, please.

Reply:
I did, but before the commit, the linter re-arranges it

Reply:
I did, but the linter re-arranges it before the commit